src/App.tsx

```diff
@@ -15,36 +17,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  ALPHABETICALLY = 'alphabetically',
+  BY_LENGTH = 'by_length',
+  REVERSE = 'reverse',
```

Comment:
Good Job! My little cooment is: You have state isReverse, you can toggle that. Why make REVERSE property? I dont get it)